subgraph/package.json

```diff
@@ -6,14 +6,14 @@
     "codegen": "npm run prepare && graph codegen",
     "build": "npm run prepare && graph build",
     "prepare": "mustache ./config/${NETWORK:-local}.json subgraph.template.yaml > subgraph.yaml",
-    "deploy-remote": "npm run codegen && npm run build && graph deploy ${NAME:-null} --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
+    "deploy-remote": "npm run codegen && npm run build && graph deploy ${NAME:-null} --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --access-token d3f08ff3f5d8463aabeb1623f9ce277c",
```

Comment:
production access token - let's never do this in the future (I mean adding it to any files that might be committed)